scripts/_run-aws-eks-commands.js

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env node
 
 const yargs = require('yargs')
-const { execSync } = require('child_process')
+const { execSync } = require('../libs/base-utils/src')
```

Comment:
OK, so we _do_ use it in non-transpiled code. Why isn't this `ts`? Should it be?